features/old/accounts/invoices.feature

```diff
@@ -36,3 +36,18 @@ Feature: Show invoices from account's page (#16015909)
     And I follow "2011-01-00000001"
     # TODO: Properly check if order is opened
     Then I should see "Add"
+
+  Scenario: Display years of invoices when there are invoices
+    Given a provider is logged in on 1st January 2011
+    And the provider is charging its buyers in prepaid mode
+    And an application plan "Fixed" of provider "foo.3scale.localhost" for 0 monthly
```

Comment:
Subscribing to applications are not needed for these tests. So, this and the following steps should not be needed.